### PR TITLE
Set descriptions for Zarr GitHub repos as soon as they're synced

### LINKED ISSUE #151: add/update stats for dandizarrs
repos under https://github.com/dandizarrs should gain description with # of files and total size similar (just without title for now) to those at https://github.com/dandisets

### PR DESCRIPTION
**Pull request by jwodder**
Closes #151.

### CODE REVIEW

File: tools/backups2datalad/manager.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+from .adataset import DatasetStats
+from .aioutil import arequest
+from .config import BackupConfig
+from .consts import USER_AGENT
+from .logging import PrefixedLogger, log
+from .util import quantify
+
+
+@dataclass
+class Manager(AsyncResource):
```
Comment (yarikoptic) [inline tools/backups2datalad/manager.py:21]: Could you please add a docstring to describe the purpose of such a manager (too generic name to discern its purpose)

**Review by yarikoptic — APPROVED** [review]
Just asked for some doc on what is the purpose of manager

File: tools/backups2datalad/manager.py (hunk 1 above)
Comment (jwodder) [reply tools/backups2datalad/manager.py:21]: Done: 041d245.

**Comment by yarikoptic** [comment]
I guess it shouldn't interfere with an already running process, so let's proceed!

**Pull request merged**